libs/transferfunction/transfer_function_widget.cpp

```diff
@@ -132,7 +132,7 @@ void TransferFunctionWidget::draw_ui()
     canvas_size.y -= 80;
```

Comment:
So the function stop compute and draw the histogram already existed? I'm a bit confused on the "scope" of this PR. 

Where are you computing the histogram? I only see the `rand()/RAND_MAX` .... Are the units the same as the axis in the transfer function?

Also the indentation in this file (although not in the code that changed) is a bit messy

Reply:
But the GUI looks very cool 😎

Reply:
I removed the rand()/RAND_MAX form the histogram initialization.
I made a mistake and some of the functions got pushed into a previous commit. I just pushed a new commit with all the changes related to this feature.